kube-runtime/src/scheduler.rs

```diff
@@ -184,11 +184,11 @@ where
     /// they are ready.
     ///
     /// The returned [`HoldUnless`] is designed to be short-lived: it has no allocations, and
-    /// no messages will be lost, even if it is reconstructed on each call to [`poll_next`].
+    /// no messages will be lost, even if it is reconstructed on each call to [`poll_next`](Self::poll_next).
     /// In fact, this is often desirable, to avoid long-lived borrows in `can_take_message`'s closure.
     ///
     /// NOTE: `can_take_message` should be considered fairly performance-sensitive, since
-    /// it will generally be executed for each pending message, for each [`poll_next`].
+    /// it will generally be executed for each pending message, for each [`poll_next`](Self::poll_next).
```

Comment:
These links were not resolved.
![image](https://user-images.githubusercontent.com/639336/103120704-be976e80-462d-11eb-8f02-cecf656f1957.png)

Reply:
Ah, good spot. I can see them as warnings in cargo fmt now with an updated nightly.